app/views/layouts/application.html.erb

```diff
@@ -42,15 +42,8 @@
         </a>
 
         <div class="header__nav-links-wrap">
-          <% if request.path_info != "/" %>
-            <%= form_tag search_path, class: search_form_class, method: :get do %>
-              <%= search_field_tag :query, params[:query], placeholder: t(".header.search_gem_html"), class: "header__search", autocomplete: "off", data: { nav_target: "search" } %>
-              <ul id="suggest" class="suggest-list"></ul>
-              <%= label_tag :query do %>
-                <span class="t-hidden"><%= t(".header.search_gem_html") %></span>
-              <% end %>
-              <%= submit_tag "⌕", id: "search_submit", name: nil, class: "header__search__icon" %>
-            <% end %>
+          <% if [root_path, advanced_search_path].exclude? request.path_info %>
```

Comment:
I'm pretty sure I have asked this before already at some older issue, but any reason to not use `current_page?` helper?

Reply:
oh, I just copied this. I think current_page? is fine. Swapped.